### ISSUE TITLE
Support VSCode using startup.jl

### ISSUE THREAD

**Issue by jonniediegelman** [BioTurboNick/AbbreviatedStackTraces.jl#38]
Currently AbbreviatedStackTraces doesn't work in VSCode if you have `using AbbreviatedStackTraces` in your startup.jl file. If you try to `using` again after startup, it still doesn't work. However it does work outside of VSCode using the startup.jl file and inside of VSCode not using the startup.jl file.

**Comment by BioTurboNick** [comment]
Can you please clarify "doesn't work"?

**Comment by jonniediegelman** [comment]
It just prints the normal stack trace as if you weren't using AbbreviatedStackTraces.

**Comment by BioTurboNick** [comment]
Hmm, so the issue is that the VSCodeServer has its own custom REPL functions. Anything that loads before it does will get overwritten.

**Comment by BioTurboNick** [comment]
It works if you don't add it to startup.jl, yes?

**Comment by BioTurboNick** [comment]
One possible workaround is to set and read an environment variable. The problem is that you'd lose the ability to display full traces on demand, as it couldn't tell the difference between an original REPL error and a general show call.

**Comment by axsk** [comment]
Another hotfix is delaying the laoding in `startup.jl`:
```julia
@async begin
    sleep(3)
    @eval using Suppressor
    @eval @suppress using AbbreviatedStackTraces
end
```

**Comment by ORBAT** [comment]
Looks like you can also do this with [`atreplinit`](https://docs.julialang.org/en/v1/stdlib/REPL/#Base.atreplinit), which I guess is a bit less hacky as it hooks directly in to REPL initialisation and guarantees VSCode's REPL stuff has had time to do its thing

```julia
atreplinit() do _
    @eval using AbbreviatedStackTraces
end
```

**Comment by BioTurboNick** [comment]
I wonder if, now that I have the requisite code in a package extension in the master branch, that AbbreviatedStackTraces in startup.jl works without extra tricks?

**Comment by BioTurboNick** [comment]
Nope, doesn't. VSCodeServer isn't defined when the extension is loaded.
